rest-service/manager_rest/rest/resources_v3_1/blueprints.py

```diff
@@ -266,6 +266,10 @@ def patch(self, blueprint_id, **kwargs):
             blueprint.description = request_dict['description']
         if 'main_file_name' in request_dict:
             blueprint.main_file_name = request_dict['main_file_name']
+        if 'creator' in request_dict:
+            check_user_action_allowed('set_owner', None, True)
+            creator = rest_utils.valid_user(request_dict['creator'])
+            blueprint.creator = creator
```

Comment:
but we already figure out the user here: https://github.com/cloudify-cosmo/cloudify-manager/blob/f2ecc3bef310d57284a1e707c19406017100f3a4/rest-service/manager_rest/rest/resources_v3_1/blueprints.py#L227-L229

..but we didn't set it before? huh? so presumably now we just need to add the `blueprint.creator = creator` line and we don't need the others?

Reply:
oh, indeed